turbo/stages/stageloop.go

```diff
@@ -40,22 +40,22 @@ func StageLoop(
 	updateHead func(ctx context.Context, head uint64, hash common.Hash, td *uint256.Int),
 	waitForDone chan struct{},
 	loopMinTime time.Duration,
+	initialCycle *bool,
```

Comment:
Since there is no need to use `initialCycle` outside of this function anymore, it can be contained within the function, as before